lib/fxpay/utils.js

```diff
@@ -119,6 +135,11 @@
     return windowRef;
   };
 
+ /**
+  * Get the App object returned from mozApps.getSelf().
+  * @param {module:utils~getAppSelfCallback} callback - the callback function.
```

Comment:
Maybe mention that the callback receives an [App object](https://developer.mozilla.org/en-US/docs/DOMApplication)